app/Services/UserGroupService.php

```diff
@@ -5,12 +5,20 @@
 namespace App\Services;
 
 use App\Contracts\UserGroupContract;
+use App\Http\Controllers\MailController;
```

Comment:
I don't think that we need to use the controller here? doesn't that go against the contract/services flow ?